src/app/components/course/course-event-create/course-event-create.component.ts

```diff
@@ -26,7 +26,6 @@ export class CourseEventCreateComponent implements OnInit {
     constructor(private route: ActivatedRoute, private courseEventService: CourseEventService, private router: Router) {
     }
 
-    // TODO: Reroute if you try and navigate to a page like this without clicking the button?
     ngOnInit(): void {
         this.getMinDate();
```

Comment:
It might be a good time to refactor EVENT_TYPES to use an API to fetch the event types. You should probably create a new action ('types') in event API and retrieve it here.